src/current/cockroachcloud/export-logs.md

```diff
@@ -377,6 +379,117 @@ Perform the following steps to enable log export from your CockroachDB {{ site.d
 {{site.data.alerts.end}}
 
+Perform the following steps to enable log export from your CockroachDB {{ site.data.products.dedicated }} cluster to Azure Monitor.
+
+1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID:
+
+	1. Visit the CockroachDB {{ site.data.products.cloud }} console [cluster page](https://cockroachlabs.cloud/clusters).
+	1. Click on the name of your cluster.
+	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`.
+
+1. [Create a Log Analytics workspace in Azure Portal](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/quick-create-workspace?tabs=azure-portal).
+
+    1. Navigate to the Log Analytics workspace. Under **Settings**, click **Agents**.
+    1. Click **Log Analytics agent instructions** section header.
+    1. In that section, note the values for **Workspace ID**, **Primary key** (or **Secondary key**)
+
+1. Use one of the following Cloud API commands to enable log export for your CockroachDB {{ site.data.products.dedicated }} cluster. The first presents a basic configuration, where all logs are sent to Azure Monitor using the default settings. The second allows for more detailed customization of the logging configuration, such as the ability to send certain log channels to specific target log tables, or the ability to redact sensitive log entries.
+
+   1. To enable log export for your CockroachDB {{ site.data.products.dedicated }} cluster with default logging configuration, issue the following Cloud API command:
+
+        {% include_cached copy-clipboard.html %}
+        ~~~shell
+        curl --request POST \
+          --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/logexport \
+          --header "Authorization: Bearer {secret_key}" \
+          --data '{"type": "AZURE_LOG_ANALYTICS", "log_name": "{log_prefix}", "auth_principal": "{workspace_id}", "azure_shared_key": "{primary_or_secondary_key}"}'
+        ~~~
+
+        Where:
+        - `{cluster_id}` is your CockroachDB {{ site.data.products.dedicated }} cluster ID as determined in step 1.
+        - `{secret_key}` is your CockroachDB {{ site.data.products.dedicated }} API key. See [API Access]({% link cockroachcloud/managing-access.md %}) for instructions on generating this key.
+        - `{log_prefix}` is a user-specified prefix for the table that will be created in the Log Analytics workspace.
+        - `{auth_principal}` is the ID for the Log Analytics workspace, **Workspace ID** from step 2.
+        - `{azure_shared_key}` is the **Primary key** or **Secondary key** from step 2.
+
+   1. To enable log export for your CockroachDB {{ site.data.products.dedicated }} cluster with custom logging configuration:
```

Comment:
I would have this as an unordered markdown list - and add an anchor link like: 

```
- <a id="custom-logging-configuration"></a>To enable log export for....
```

Reply:
Then the `a,b,c` procedure for the custom configuration will not be nested under `b`.

Reply:
Added Custom heading. Modified so unordered list is rendered correctly with bullets.